docs/Makefile

```diff
@@ -0,0 +1,21 @@
+export HUGO_VERSION?=0.85.0
```

Comment:
cant we move this makefile into the /make folder so we have everything together?

Reply:
I'd really would go for optimizations on follow-ups as all of this can be handled much better (we can even have hugo inside the repo.. so versions are tight and we treat it like all the other CI deps)